src/components/App.jsx

```diff
@@ -34,7 +38,27 @@ const KeyboardAwareSidebar = ({ children }) => {
 
 const App = props => {
   const { showBottomNav, settingsCollection } = props
+  const { t } = useI18n()
+  const client = useClient()
   const settings = getDefaultedSettingsFromCollection(settingsCollection)
+
+  const showAlert = t => {
+    Alerter.error(t('Error.fetch-error'), {
+      buttonText: t('General.reload'),
+      buttonAction: () => window.location.reload()
+    })
+  }
+
+  useEffect(() => {
+    const onError = e => {
+      if (!getActivatePouch() && hasFetchFailedError(e)) {
+        throttle(showAlert(t), 300)
+      }
+    }
+
+    client.setOnError(onError)
+  }, [client, t])
```

Comment:
If `t` changes, we'll call again the `setOnError` method and that'll throw an error. Why not just running this setOnError with an empty dep's array?

Reply:
Yes absolutely